src/MagnumPlugins/GltfImporter/Test/GltfImporterTest.cpp

```diff
@@ -4492,6 +4495,53 @@ void GltfImporterTest::meshUnorderedAttributes() {
     CORRADE_COMPARE(mesh->attributeFormat(customAttribute1), VertexFormat::Vector3);
 }
 
+void GltfImporterTest::meshMorphTargetAttributes() {
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("GltfImporter");
+
+    CORRADE_VERIFY(importer->openFile(Utility::Path::join(GLTFIMPORTER_TEST_DIR, "mesh-morph-target-attributes.gltf")));
+    CORRADE_COMPARE(importer->meshCount(), 1);
+
+    Containers::Optional<Trade::MeshData> mesh = importer->mesh(0);
+    CORRADE_VERIFY(mesh);
+    CORRADE_COMPARE(mesh->attributeCount(), 6);
+
+    /* Base mesh (position, normal and color) */
+    CORRADE_COMPARE(mesh->attributeCount(-1), 2);
+
+    CORRADE_VERIFY(mesh->hasAttribute(MeshAttribute::Position));
```

Comment:
It should be enough to just compare the `attributeCount()`, the `hasAttribute()` is internally just `return attributeCount(name) > 0;`.